src/main/java/wooteco/subway/line/domain/Line.java

```diff
@@ -47,6 +47,9 @@ public Sections getSections() {
         return sections;
     }
 
+    public List<Section> sections() {
+        return sections.getSections();
+    }
     public void update(Line line) {
```

Comment:
```suggestion
    public List<Section> sections() {
        return sections.getSections();
    }

    public void update(Line line) {
```

아래 메서드는 사용되지 않고 있네요. 안 쓰는 코드는 바로바로 지워 주세요!